### PR TITLE
Update common-labels.md

### PR DESCRIPTION
**Pull request by xfc**
To fix a typo.

### CODE REVIEW

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
I think the sentence could do with rephrasing to a simpler format since it currently sounds ambiguous. Would you be up to it @xfc ?

**Comment by divya-mohan0209** [comment]
Also, please sign the CLA before any further commits are pushed @xfc

**Comment by xfc** [comment]
> I think the sentence could do with rephrasing to a simpler format since it currently sounds ambiguous. Would you be up to it @xfc ?

Hi @divya-mohan0209 , thanks for the feedback. I just signed the CLA. 
The meaning of the sentence seems clear enough for me. A rephrasing could make it wordy.

File: content/en/docs/concepts/overview/working-with-objects/common-labels.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ in some cases, the same namespace. For example, WordPress can be installed more
 than once where different websites are different installations of WordPress.
 
 The name of an application and the instance name are recorded separately. For
-example, WordPress has a `app.kubernetes.io/name` of `wordpress` while it has
+example, WordPress has an `app.kubernetes.io/name` of `wordpress` while it has
```
Comment (divya-mohan0209) [inline content/en/docs/concepts/overview/working-with-objects/common-labels.md:68]:
```suggestion
example, WordPress has an application name, represented as `app.kubernetes.io/name` with a value of `wordpress` while it has
```

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
For better clarity.

**Comment by divya-mohan0209** [comment]
@xfc : I would try making it as idiomatic as possible at the expense of wordiness because merely stating the below could confuse a new user. I've suggested a rephrasing incorporating your suggestion.

> WordPress has an `app.kubernetes.io/name` of `wordpress`

**Comment by tengqm** [comment]
does this fall into trivial edits category?

**Comment by divya-mohan0209** [comment]
Yep, it does @tengqm

**Comment by xfc** [comment]
I'm not sure whether this falls to the trivial edits category, what I do know is this is an error.
I heard the story that Donald Kruth paid real money for every person who spotted any typo in his book. I like the Kubernetes's documention and would like it to be as perfect as possible, (or as perfect as Kruth's books).

**Comment by tengqm** [comment]
@xfc Would you like revise the text as per suggested by Divya?

File: content/en/docs/concepts/overview/working-with-objects/common-labels.md (hunk 1 above)
Comment (kbhawkey) [inline content/en/docs/concepts/overview/working-with-objects/common-labels.md:68]: Hello @xfc .
You could accept the suggested changes, or clarify the text a bit more.

**Comment by natalisucks** [comment]
Hi @xfc – if you could please review the changes as proposed by Divya and accept them, or add your own clarification that improves on your initial change, then I'd be happy to approve this PR. Thanks!

**Comment by kbhawkey** [comment]
Hello @xfc .
I am closing this pull request. It is okay to reopen the PR and continue to work on the changes.
/close

**Pull request closed without merging**